yb-voyager/cmd/importSchemaYugabyteDB.go

```diff
@@ -90,6 +113,20 @@ func executeSqlFile(file string, objType string, skipFn func(string, string) boo
 	return nil
 }
 
+func shouldSkipDDL(stmt string) (bool, error) {
+	skipReplicaIdentity := strings.Contains(stmt, "ALTER TABLE") && strings.Contains(stmt, "REPLICA IDENTITY")
```

Comment:
you can return here if this comes out to be true. No need for further check, which is a bit heavy involves parsing and all...